src/client/src/ui/spotTile/components/PriceMovement.tsx

```diff
@@ -10,28 +10,27 @@ interface Props {
 
 const MovementIcon = styled('i')<{ show: boolean; color: string }>`
   text-align: center;
-  color: ${({ theme: { palette }, color }) => palette[color].normal}}};
+  color: ${({ theme, color }) => theme[color].base}}};
   visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
 `
 
 const MovementValue = styled('div')`
   font-size: 11px;
-  color: ${({ theme: { text } }) => text.textMeta};
+  opacity: 0.59;
 `
 
-//TODO: SVG icons
 const PriceMovement: React.SFC<Props> = ({ priceMovementType, spread }) => (
   <Flex alignItems="center" justifyContent="center" direction="column" width="100%">
     <MovementIcon
       show={priceMovementType === PriceMovementTypes.Up}
-      color="accentGood"
+      color="good"
```

Comment:
I think `intent` or `accent` might read a little more coherently — personally I think the color of `good` is `purple` 😛